src/components/boardmember/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './style.less';
+
+interface BoardMemberProps {
+  name: string;
+  title: string;
+  photo: string;
+}
+
+const BoardMember: React.FC<BoardMemberProps> = ({ name, title, photo, }) => (
+  <div className="board-member-card">
+      <div className="board-member-photo" style={{backgroundImage: 'url('+photo+')'}} />
+      <div className="board-member-info">
+          <div className="board-member-title">
+            {name}
+          </div>
+          <div className="board-member-description">
```

Comment:
Should be styled on the Figma the name is larger than the role + a larger font weight as well 
![image](https://user-images.githubusercontent.com/8893880/80935495-bda18c00-8d81-11ea-975a-40fc623ba1c3.png)

Reply:
I made the text bolder and changed the fonts slightly for the name and role - I couldn't really see the exact font sizes in the Figma so I did 1.1 em and 0.8em

Reply:
To increase the size of the name I made it 1.2 rem and added a margin-bottom of 5px 
![image](https://user-images.githubusercontent.com/8893880/81145521-44de3380-8f2b-11ea-80d4-5e371930d55b.png)